data/resistance intro.txt

```diff
@@ -0,0 +1,1187 @@
+			choice
+				`	"So, you're the 'Resistance', but what exactly are you resisting?"`
+					goto resist
+				`	"Is there any meaning to the destinations of the deliveries, or do you just hide what I bring you wherever you can?"`
+					goto deliveries
+				`	"With all I've been bringing, do you plan to launch some kind of assault on the Heliarch somewhere?"`
+					goto assault
+				`	"I don't have any more questions, what's that about some job you had for me?"`
+					goto mission
+			label assault
+			`	"Doomed, we are, if a head-on battle, we seek," the Saryd says.`
+			`	"Too unbalanced, our forces are. Strong, Heliarch ships are. To take a major shipyard, our absolute priority would be, if to stand a chance in conflict, we were," one of the Kimek further adds.`
+			choice
+				`	"So, you're the 'Resistance', but what exactly are you resisting?"`
+					goto resist
+				`	"Is there any meaning to the destinations of the deliveries, or do you just hide what I bring you wherever you can?"`
+					goto deliveries
+				`	"Am I the only outsider helping the Resistance?"`
+					goto outsiders
+				`	"I don't have any more questions, what's that about some job you had for me?"`
+					goto mission
+			label mission
+			`	They give you a brief rundown of their activities, for context, then the Saryd says, "Though at peace, we are, still waging war, in their minds, the Heliarch are. For long, banished, the Quarg have been. The several millennia old grudge, only by the Heliarchs, sustained is."`
+			`	"Against their presence, we, the people, are not. Made known to them, this must be," the Kimek beside you says.`
+			`	Finally, the Arachi hands you a small data chip, saying, "Details of our continued efforts, in this we have recorded. To deliver this message, we ask you, Captain <last>."`
+			choice
+				`	"You really think the Quarg are just gonna come and help just because you asked?"`
+					goto unlikelyhelp
+				`	"Alright, I'll take this to <destination>."`
+					accept
+				`	"I'm a bit busy here, can I come back pick it up some other day?"`
+					goto deferchip
+			label unlikelyhelp
+			`	"Naive, we are not," the Saryd says. "Cautious, the Quarg are. To send a triumphant war fleet, the last thing we expect of them is."`
+			`	"Still, great knowledge, the Quarg posess. More than the Heliarch, even," the Kimek beside you continues. "Immediate help, we are seeking not. Help they can, in other ways."`
+			choice
+				`	"Alright, I'll take this to <destination>."`
+					accept
+				`	"I'm a bit busy here, can I come back pick it up some other day?"`
+					goto deferchip
+			label deferchip
+			`	The group isn't terribly excited about your lack of enthusiasm, as the Kimek discuss among themselves for a bit.`
+			`	After it all, the Kimek beside you reactivates the translation box, saying, "Dangerous, it would be, if found by the Heliarchs, this chip was. When ready to leave Coalition space, you are, return here, you must, to pick it up."`
```

Comment:
```suggestion
			`	Afterwards, the Kimek beside you reactivates the translation box, saying, "Dangerous, it would be, if found by the Heliarchs, this chip was. When ready to leave Coalition space, you are, return here, you must, to pick it up."`
```